students/Chao/session01/codingBat/Warmup-1.py

```diff
@@ -0,0 +1,56 @@
+def sleep_in(weekday, vacation):
+    return(not weekday or vacation)
+
+def monkey_trouble(a_smile, b_smile):
+    return(not(a_smile ^ b_smile))
```

Comment:
note that `^` is "bitwise" or -- it will work pretty much the same for booleans, but not for anything else. And it shortcuts differently.

you generally want "or" if you aren't purposely manipulating bits.